tools/hifiasm/hifiasm.xml

```diff
@@ -117,8 +117,8 @@
         #end if
 
         #if str($hic_partition.hic_partition_selector) == 'set':
-            --h1 ${ ' '.join(["'%s'" % $x for $x in $hic1_inputs]) }
-            --h2 ${ ' '.join(["'%s'" % $x for $x in $hic2_inputs]) }
+            --h1 ${ ','.join(["%s" % $x for $x in $hic1_inputs]) }
```

Comment:
why do you remove the quotes? They still need to be quoted, correct?

Maybe you want to add a test, you can simply add the same two files in your latest test I think.

Reply:
ah I took the quotes out to it would match with the example in the hifiasm help (`r1.fq,r2.fq` instead of `'r1.fq','r2.fq'`), but I can put them back in if that is better practice?

and yes I'll add a test 👍🏼

Reply:
imaging your data is stored on a path that has spaces in its names, like "/home/stupid bjoern/foo/bar" ... if you do not use single quotes on bash if you specify this path, every tool will interpret it as two paths... that why we need to have everything single quotes ... because we never know if people run those tools on a path with "spaces in its path".

Reply:
ohhh okay, I understand now, sorry about that! I'll put them back 👍🏼